pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1783,7 +1869,7 @@ def create_model(df_sets, df_parameters, default={}):
                 from_units=model.user_units["currency_volume"],
                 to_units=model.model_units["currency_volume"],
             )
-            for key, value in model.df_parameters["PipelineOperationalCost"].items()
+            for key, value in _df_parameters.items()
```

Comment:
So, if I am reading the code correctly, if `config.hydraulics` is not `Hydraulics.post_process`, then all of the values for `model.p_pi_Pipeline` will be 0.0001 (see line 1762), and anything in the "PipelineOperationalCost" tab will be ignored. Is that the desired behavior? That doesn't seem like what we should want.

Reply:
This may be why the objective changes slightly in the tests. All piping costs may be set to the default value instead of reading from input. See my comment at 1762.

Reply:
the objective values is restored

Reply:
Resolving.